3rdparty/libprocess/include/process/protobuf.hpp

```diff
@@ -39,8 +39,11 @@ inline void post(const process::UPID& to,
                  const google::protobuf::Message& message)
 {
   std::string data;
-  message.SerializeToString(&data);
-  post(to, message.GetTypeName(), data.data(), data.size());
+  if (message.SerializeToString(&data)) {
+    post(to, message.GetTypeName(), data.data(), data.size());
+  } else {
+    LOG(WARNING) << "Error serialising: " << message.DebugString();
```

Comment:
I don't think we can log the message here since it may be huge (e.g. 2GB), how about an ERROR level message like:

> Failed to post 'UpdateSlaveMessage' to slave@192.168.0.1:5050: Failed to serialize

(you can just `<< to` to log the destination, and `m.GetTypeName()` for the name).

Reply:
Done.